### PR TITLE
Updated "Create FHIR Data" to use TS Client

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/docs/docs/tutorials/api-basics/create-fhir-data.md [hunk 1]
```diff
@@ -171,132 +128,83 @@ To get this to be linked up, you'll need to have the identifiers for the Patient
...
-const [reportId] = await createReport(serviceRequestId, patientId);
+```
+
+### Create Report
+
+```ts
+import { DiagnosticReport } from '@medplum/fhirtypes';
+
+const reportData: DiagnosticReport = {
+  resourceType: 'DiagnosticReport',
```
Comment (codyebberson) [inline packages/docs/docs/tutorials/api-basics/create-fhir-data.md:194]: `status` is a required field

File: packages/docs/docs/tutorials/api-basics/create-fhir-data.md [hunk 2]
```diff
@@ -171,132 +128,83 @@ To get this to be linked up, you'll need to have the identifiers for the Patient
...
-        // Create the second Observation resource.
+### Create the Observations
+
+```ts
+import { Observation } from '@medplum/fhirtypes';
+
+// Create two observations from the array
+const observationData: Observation[] = [
+  {
+    resourceType: 'Observation',
```
Comment (codyebberson) [inline packages/docs/docs/tutorials/api-basics/create-fhir-data.md:139]: Add `status`

File: packages/docs/docs/tutorials/api-basics/create-fhir-data.md [hunk 3]
```diff
@@ -171,132 +128,83 @@ To get this to be linked up, you'll need to have the identifiers for the Patient
...
+    },
+    valueQuantity: {
+      value: 5.7,
+      unit: 'mg/dL',
+      system: 'http://unitsofmeasure.org',
+      code: 'mg/dL',
+    },
+  },
+  {
+    resourceType: 'Observation',
```
Comment (codyebberson) [inline packages/docs/docs/tutorials/api-basics/create-fhir-data.md:159]: Add `status`

**Review by codyebberson — CHANGES_REQUESTED** [review]
Let's add the required fields to prepare for strict mode

File: packages/docs/docs/tutorials/api-basics/create-fhir-data.md (hunk 1 above)
Comment (rahul1) [reply packages/docs/docs/tutorials/api-basics/create-fhir-data.md:194]: Good catch

**Review by codyebberson — APPROVED** [review]

**Pull request merged**